server/routes/routes.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+
+const router = express.Router();
+
+router.get('/restaurants/:restaurantId/reviews', (req, res) => {
```

Comment:
consider refactoring your router methods to all chain off of the router const like we did on our TA e.g.: 
router 
  .get(...)
  .post(..)
  etc..